### ISSUE TITLE
Daycore reduces the animations framerate

### ISSUE THREAD

**Issue by zXd12** [ppy/osu#24600]
### Type

Game behaviour

### Bug description

It's very noticeable in mania, but it affects all game mods (probably. the sliders seems to be less smooth in standard but I'm not sure). It is daycore related, half time doesnt do that when activated.

The speed modifier doesn't change anything, the framerate decrease is the same at 0.99x and 0.5x speed. Changing the framerate limit and scree mode didn't change anything either.

This is not a general performance issue, my cursor is still perfectly smooth and the fps graphs are equaly flat with and without daycore.

### Version

2023.085.0

### Logs

[updater.log](https://github.com/ppy/osu/files/12389331/updater.log)
[database.log](https://github.com/ppy/osu/files/12389332/database.log)
[input.log](https://github.com/ppy/osu/files/12389333/input.log)
[network.log](https://github.com/ppy/osu/files/12389334/network.log)
[performance.log](https://github.com/ppy/osu/files/12389335/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12389336/runtime.log)

**Comment by peppy** [comment]
Are you watching a replay when observing this? It sounds like interpolation isn't running correctly for you. I've tested and can't repro this, so might be something local to your system (although logs don't reveal any hints to this happening).

**Comment by zXd12** [comment]
No, it happens during gameplay. I have had problems with being on the dev branch of windows with other games but I recently switched to a preview build on the same hardware so this shouldn't be the problem. The weird thing is that I have this problem since a long time but sometimes it only starts happening after playing the game for a while. I'll try to test things to find what exactly causes the problem and I'll reopen this if I find anything.

**Comment by zXd12** [comment]
I have found a fix for my problem, but even more confusing elements.

I am using bluetooth earbuds (Jabra Elite Active 75t) every time this happens, and when I switch to a usb headphone the animation framerate fixes itself. Sometimes the animation framerate will be fine on every map, even if I use my earbuds.
I'm pretty sure it has to do with audio output handling, I can litteraly see the framerate fix or break itself when I change the device in the settings while a beatmap plays.

When the problem happens, it affects Most of the maps but not all of them.
I will try to reproduce my tests with the same set of maps another day for consistency, but for now I have this :
were affected :
- 1714520 Camellia - Final Flash Flight
- 1423058 Street - Hestia
- 988031 Artisan - Summon
were not affected :
- 846536 Blank Banshee - B__Start Up
- 1276332 ARForest - The Last Page
- 1895281 Silentroom - Protoflicker
This is valid for every difficulty of each map. I wasn't able to see a difference in their file that could be the source of the problem tho.

This is not a daycore specific problem, it has to do with audio stretching. Wind up/down also have this problem, but the framerate diminution varies depending of the playrate. From my testing 85% seems to be one of the worst spot.

https://github.com/ppy/osu/assets/43031345/8e7d9e63-844b-432a-a50d-a5ddbb77ca15

Sometimes the problem fixes itself between game reboots. I relaunched the game to reproduce it because my first screen recording was to large to fit on github, but the it had disapeared.

I don't think this is really a high priority bug to fix, since I seem to be the only one to have this problem and found a way to fix it. (Plus I am probably the only player who uses a bluetooth audio output anyway, my offset is litteraly -300ms, I can't even use hitsounds)
